src/VonageChannelServiceProvider.php

```diff
@@ -22,8 +22,6 @@ public function register()
         $this->app->singleton(Client::class, function ($app) {
             $config = $app['config']['vonage'];
 
-            $httpClient = null;
```

Comment:
This line was unused always since the variable is being initialised in the very next line.

This change is not related to the new feature introduced .